modules/meshrenderinggl/include/modules/meshrenderinggl/datastructures/rasterization.h

```diff
@@ -67,6 +96,18 @@ class IVW_MODULE_MESHRENDERINGGL_API Rasterization {
      * @return Specific information about this rasterization type/instance.
      */
     virtual Document getInfo() const;
+
+    /**
+     * \brief Get a copy of the object. Ownership goes to the caller.
+     * @return A copy with the same data and type as the original.
+     */
+    virtual Rasterization* copy() const = 0;
+
+    /**
+     * \brief Mot simple spatial 3D data object, hosting a model and world matrix.
+     * Should be applied to geometry, and then set as a uniform.
+     */
+    rasterization::SimpleSpatialEntity spatialTransformation_;

```

Comment:
Is it not a bit backwards to have a transform in the rasterizer? 
This is equivalent to a global transformation?

Reply:
its per rasterisation object so it's not global. Many rasterisations are combined in the final rendering. each with individual transforms.

Reply:
This PR is mostly in response to [this pull request](https://github.com/inviwo/inviwo/pull/931), looking for a way of changing transforms without copying the (rather large) data objects. Adding the transform as a property for rendering only solves that.